src/components/service-viewer/service-viewer.vue

```diff
@@ -168,7 +170,7 @@ export default {
       let allLayers;
       let layer;
       let extent;
-      // TODO same getCapabilities request has different format in the response (Tredd or Geoserver)
+      // NOTE same getCapabilities request has different format in the response (Thredd or Geoserver)
```

Comment:
Oh haha here you changed it to NOTE, see comment above :) still, whatever you prefer